### ISSUE TITLE
Screen reader does not announce the label and required information on Description (Rich text editor) in windows.

### ISSUE THREAD

**Issue by msftedad** [ckeditor/ckeditor5#19136]
**Test Environment:**
OS: Windows 11 24H2 (OS Build 26100.6584)
Browser: Edge (Version 140.0.3485.81)
Screen Reader: NVDA (2025.2)
URL: https://codesandbox.io/p/sandbox/beautiful-hamilton-2zxd9l

**User Impact:**
Visually impaired users relying on screen readers may miss critical information if labels and required fields are not properly announced, leading to incomplete understanding and potential task failure.

**Repro Steps:**
1. Open the URL: https://codesandbox.io/p/sandbox/beautiful-hamilton-2zxd9l in windows.
2. Press tab key to navigate to the 'Description (Rich text editor)'.
3. Verify whether screen reader announces the label and required information on Description (Rich text editor) in windows or not.

**Actual Result:**
Screen reader does not announce the label and required information on Description (Rich text editor) in windows.
Screen reader announces: Rich text editor, main edit multi line blank

**Refer Attachment:**
1. Screen reader does not announce the label and required information on Description (Rich text editor) in windows.png
2. Screen reader does not announce the label and required information on Description (Rich text editor) in windows.mp4

https://github.com/user-attachments/assets/9513681f-99e2-4d2c-a427-726ad7fb9453
<img width="960" height="533" alt="Image" src="https://github.com/user-attachments/assets/c1b274a3-ef59-447d-99ba-252e6488d43b" />

**Expected Result:**
Screen reader should announce the label and required information on Description (Rich text editor) in windows.

**Comment by Witoso** [comment]
This is CKEditor 5 v27.0.1. We are right now at version 46, where this mechanism should work correctly.

**Comment by msftedad** [comment]
@Witoso,

Currently, SDM is using CKEditor 5 version 43.1.1.
There is no usage of version 27.0.1 anywhere in the application.

I have upgraded CKEditor to the latest version (47.3.0). However, after upgrading, we are encountering an issue where the CKEditor license key is missing, and as a result, the editor is not functioning as expected.

As discussed with the FTE team, we would like clarification on the following:

Is it mandatory to upgrade specifically to version 46.x, or
Will upgrading directly to the latest version resolve the license key issue?

Kindly review and let us know the recommended approach to resolve this issue.

Could you please reopen this issue?

**Comment by Witoso** [comment]
> Currently, SDM is using CKEditor 5 version 43.1.1.

The provided code example, however, appears to be based on build version v27, which makes it difficult to reliably reproduce the issue.

You mentioned upgrading CKEditor to the latest version (**47.3.0**). After the upgrade, the editor is not working because the license key is missing. The current CKEditor version requires a license key to be explicitly provided, either the `GPL` key (as described in the [documentation](https://ckeditor.com/docs/ckeditor5/latest/getting-started/licensing/license-key-and-activation.html#using-the-gpl-key)) or a valid commercial license.

I’ll be happy to reopen it once a test case using the current editor version, along with clear reproduction steps, is provided.

If you need direct assistance in the meantime, please feel free to [contact our support team](https://ckeditor.com/contact/).
